pkg/integrations/v4/supervisor_fb.go

```diff
@@ -98,6 +106,17 @@ func buildFbExecutor(fbIntCfg FBSupervisorConfig, cfgLoader *logs.CfgLoader) fun
 			return nil, errors.Wrap(err, "failed to create temporary fb sFBLogger config file")
 		}
 
+		if fbConfigTempFiles, err := removeFbConfigTempFiles(MaxNumberOfFbConfigTempFiles); err != nil || fbConfigTempFiles != nil {
+
+			for _, file := range fbConfigTempFiles {
+				log.Debugf("Removed %s config temp file.", file)
+			}
+
+			if err != nil {
+				log.WithError(err).Warn("Failed removing config temp files.")
+			}
+		}
```

Comment:
Great thing that you allow partial failure 🙇 . In that case, I'd suggest renaming `fbConfigTempFiles` to `removedConfigTempFiles`, because I confess that I initially thought that this method was returning all the files that it _attempted_ to remove, instead of the ones that it had actually removed.

Reply:
Totally!